reports/Arkansas.md

```diff
@@ -1,19 +1,28 @@
 ## Bentonville
 
-### Law enforcement gas a crowd chanting “we want peace” right after exiting the building. | June 1st
+### Law enforcement tear-gas a crowd chanting “we want peace” right after exiting the building. | June 1st
 
-Law enforcement gas a crowd chanting “we want peace” right after exiting the building, in the video there was no order to disperse, the crowd seemed to be peacefully chanting.
-Article says crowd started throwing water bottles and firecrackers. Sheriff's office claims rocks were thrown.
+A crowd gathered on the Bentonville Square to protest. Around 9 PM, law enforcement gassed a crowd chanting “we want peace.” Local news says some people throwing water bottles and firecrackers. Police also claim rocks were thrown. Police claim that at least one police vehicle was damaged. Local news reports vary -- some reports have protesters "kicking" a police vehicle; some televised reports have protesters "damaging" a police vehicle; most written articles merely report that police say protesters damaged a police vehicle. Assuming that a police vehicle was in fact damaged, it is unclear to what extent (though per one op-ed writer, the police chief says $8000 worth of damage was done).
```

Comment:
I think

> Assuming that a police vehicle was in fact damaged, it is unclear to what extent (though per one op-ed writer, the police chief says $8000 worth of damage was done).

Can be simplified to just "The police chief alleges $8,000 worth of damage was done"

Reply:
I'm somewhat reluctant to reference information from an op ed as fact.

Reply:
Okay, then I think maybe it's better to just remove this sentence. I think you cover the ambiguity plenty in the preceding one